org-roam-refile.el

```diff
@@ -30,9 +30,70 @@
 ;; Org-roam refile allows you to refile notes to your nodes.
 ;;
 ;;; Code:
+(eval-when-compile
+  (require 'org-roam-macs)
+  (require 'org-macs))
+
 (defvar org-auto-align-tags)
 (defvar org-loop-over-headlines-in-active-region)
 
+(defun org-roam--file-keyword-get (keyword)
```

Comment:
These should probably go into `org-roam-utils`.

Reply:
Agreed. I was trying to keep all the refile-specific functions in `org-roam-refile.el` until you're ready to add that to the requires at the top of `org-roam.el`.